Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/JavaArrayList.java

```diff
@@ -24,19 +24,19 @@ public static void main(String[] args) {
 
     }
 
-    private static ArrayList<ArrayList<Integer>> createTable(int rowsNumber) {
-        ArrayList<ArrayList<Integer>> table = new ArrayList<>(rowsNumber);
+    private static List<List<Integer>> createTable(int rowsNumber) {
+        List<List<Integer>> table = new ArrayList<>(rowsNumber);
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...